src/platform/packages/private/kbn-apm-config-loader/src/config.ts

```diff
@@ -22,6 +22,7 @@ import type { ApmConfigSchema } from './apm_config';
 // https://www.elastic.co/guide/en/apm/agent/nodejs/current/configuration.html
 const DEFAULT_CONFIG: AgentConfigOptions = {
   active: true,
+  captureHeaders: false,
```

Comment:
Are there docs/tutorials that need to be updated for customers that ship APM data? Just to guide them how to capture/configure headers if they want to.